content/agent/_index.md

```diff
@@ -64,7 +64,13 @@ To automatically transition between agent configuration paths and formats from A
 
 `sudo -u dd-agent -- datadog-agent import`
 
-The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
+The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.  
```

Comment:
The command parses an existing `datadog.conf` and converts all bits the new Agent supports into the `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.

Reply:
👌

Reply:
is "all the bits" the right phrase here?

Reply:
s/bits/configuration options/ does sound cleaner.

Reply:
👌